cdap-common/bin/cdap

```diff
@@ -74,6 +74,7 @@ case ${1} in
   upgrade) shift; cdap_upgrade_tool ${@}; __ret=${?} ;;
   run) shift; cdap_run_class ${@}; __ret=${?} ;;
   sdk) shift; cdap_sdk ${@}; __ret=${?} ;;
+  setup) shift; cdap_setup_tool ${@}; __ret=${?} ;;
```

Comment:
This name is really ambiguous, especially given this is a very specific use case. Maybe `hbase_setup` or something, instead? This makes it seem like it's used for more than just adding coprocessors to HBase.